packages/web-twig/src/Resources/components/Select/README.md

```diff
@@ -0,0 +1,61 @@
+| `helperText`      | `string`                                                             | `null`           | no       | Custom helper text                                                                                                                 |
+| `id`              | `string`                                                             | —                | yes      | Input and label identification                                                                                                     |
+| `iconName`        | `string`                                                             | `'chevron-down'` | no       | Type of arrow icon                                                                                                                 |
+| `isDisabled`      | `bool`                                                               | `false`          | no       | If true, input is disabled                                                                                                         |
+| `isFluid`         | `bool`                                                               | `false`          | no       | If true, the element spans to the full width of its parent                                                                         |
+| `isLabelHidden`   | `bool`                                                               | `false`          | no       | If true, label is hidden                                                                                                           |
+| `isRequired`      | `bool`                                                               | `false`          | no       | If true, input is required                                                                                                         |
+| `label`           | `string`                                                             | —                | yes      | Label text                                                                                                                         |
+| `validationText`  | `string`, `string[]`                                                 | `null`           | no       | Validation text                                                                                                                    |
+| `name`            | `string`                                                             | `null`           | no       | Input name                                                                                                                         |
+| `validationState` | [Validation dictionary][dictionary-validation], `error` (deprecated) | `null`           | no       | Type of validation state. [**DEPRECATED**][deprecated] The value "error" in the dictionary will be replaced by the value "danger". |
```

Comment:
do not add deprecated values please. Remove error.

Reply:
done